### PR TITLE
pod dependencies

### PR DESCRIPTION
**Pull request by aboedo**
moves the iOS dependencies away from the script that downloads and copies over the files, and into pods. 
This means no more duplicated files in the repos, and super easy updating of dependencies. It also unifies the system with android, which already used the play services resolver.

### CODE REVIEW

File: Subtester/Assets/Scenes/Main.unity [hunk 1]
```diff
@@ -227,14 +227,12 @@ MonoBehaviour:
   m_GameObject: {fileID: 559298752}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: 4364eb85809d4409db551df88f4078c1, type: 3}
```
Comment (aboedo) [inline Subtester/Assets/Scenes/Main.unity:230]: I just saved and committed changes in the editor in hopes that the subtester app will work for others without having to re-attach objects. don't know if it'll work ¯\\_(ツ)_/¯

File: RevenueCat/Plugins/Editor/RevenueCatDependencies.xml [hunk 2]
```diff
@@ -4,4 +4,7 @@
         <androidPackage spec="org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.2.61" />
         <androidPackage spec="com.revenuecat.purchases:purchases:3.1.0" />
     </androidPackages>
+    <iosPods>
+        <iosPod name="PurchasesHybridCommon" version="1.1.0" minTargetSdk="9.0"/>
+    </iosPods>
```
Comment (aboedo) [inline RevenueCat/Plugins/Editor/RevenueCatDependencies.xml:9]: This is the bulk of the changes

File: Subtester/ProjectSettings/GvhProjectSettings.xml [hunk 3]
```diff
@@ -1,6 +1,10 @@
 <projectSettings>
   <projectSetting name="com.google.unity.oss.play_services_resolverAnalyticsCookie" value="aedf5e7251b14322af19f45bfbff1d9a" />
   <projectSetting name="com.google.unity.oss.play_services_resolverAnalyticsEnabled" value="True" />
+  <projectSetting name="Google.IOSResolver.AutoPodToolInstallInEditor" value="False" />
+  <projectSetting name="Google.IOSResolver.CocoapodsIntegrationMethod" value="2" />
+  <projectSetting name="Google.IOSResolver.PodfileEnabled" value="True" />
+  <projectSetting name="Google.IOSResolver.PodToolExecutionViaShellEnabled" value="True" />
```
Comment (aboedo) [inline Subtester/ProjectSettings/GvhProjectSettings.xml:7]: cocoapods config for the play services resolver

File: RevenueCat/Plugins/iOS/PurchasesUnityHelper.m [hunk 4]
```diff
@@ -7,11 +7,7 @@
 #import <Foundation/Foundation.h>
 #import <StoreKit/StoreKit.h>
 #import <AdSupport/AdSupport.h>
-#import "RCPurchases.h"
-#import "RCPurchaserInfo.h"
-#import "RCCommonFunctionality.h"
-#import "RCPurchaserInfo+HybridAdditions.h"
-#import "RCErrorContainer.h"
+#import <PurchasesHybridCommon/PurchasesHybridCommon.h>
```
Comment (aboedo) [inline RevenueCat/Plugins/iOS/PurchasesUnityHelper.m:10]: the files are part of the framework now

**Comment by aboedo** [comment]
@vegaro bump

File: Subtester/Assets/Scenes/Main.unity (hunk 1 above)
Comment (vegaro) [reply Subtester/Assets/Scenes/Main.unity:230]: hopefully it works, I have tried before but without success

**Pull request merged**